scalafix-testkit/src/main/scala/scalafix/testkit/SemanticRuleSuite.scala

```diff
@@ -95,7 +96,7 @@ abstract class SemanticRuleSuite(
       val key = lintMessages.head.category.key(rule.name)
       val explanation =
         s"""|To fix this problem, suffix the culprit lines with
-            |   // scalafix: $key
+            |   // assert: $key
```

Comment:
how does the `// assert` comment work?

Reply:
You can assert which linter message is reported at which specfic lines https://github.com/scalacenter/scalafix/blob/master/scalafix-tests/input/src/main/scala/test/NoInfer.scala#L7

The test fails if
- the linter category doesn't match,
- a message is not reported on that line
- a message is reported at a line without an assert

Reply:
Nice! This is only for testkit right?

Reply:
Yes, this is only testkit. Escape hatches for users are yet unimplemented https://github.com/scalacenter/scalafix/issues/241